rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/rmnode/RMDeployingNode.java

```diff
@@ -80,6 +87,13 @@ public RMDeployingNode(String name, NodeSource ns, String command, Client provid
         this.description = description;
     }
 
+    public RMDeployingNode(String name, NodeSource ns, String command, String description, Client provider,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 8 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)